nix/installer.nix

```diff
@@ -25,11 +25,58 @@ in
   documentation.enable = false;
   documentation.man.man-db.enable = false;
 
+  # reduce closure size through package set crafting
+  # where there's no otherwise globally effective
+  # config setting available
+  # TODO: some are candidates for a long-term upstream solution
+  nixpkgs.overlays = [
+    (final: prev: {
+      # save ~12MB by not bundling manpages
+      coreutils-full = prev.coreutils;
```

Comment:
Also question here: Do we actually care about this? We are not shipping manpages afaik and they are hopefully in a separate output?

Reply:
> they are hopefully in a separate output?

Unfortunately not, at least in nixos-23.11:
```nix
  postInstall = optionalString (isCross && !minimal) ''
    rm $out/share/man/man1/*
    cp ${buildPackages.coreutils-full}/share/man/man1/* $out/share/man/man1
  ''
  # du: 8.7 M locale + 0.4 M man pages
  + optionalString minimal ''
    rm -r "$out/share"
  '';
```
`minimal` is `false` on `-full`